setup.sh

```diff
@@ -5,20 +5,29 @@ then
 fi
 
 MODULE_NAME=$1
-if ! [[ "$MODULE_NAME" =~ ^.*\/conduit-connector-(.*)$ ]]
+if ! [[ "$MODULE_NAME" =~ ^github.com\/.*\/conduit-connector-(.*)$ ]]
 then
-  echo "Module name '$MODULE_NAME' not in required format 'example/conduit-connector-connectorname'"
-  exit 1
+  echo "Module name ${MODULE_NAME} not in recommended format github.com/repository/conduit-connector-connectorname.\n"
```

Comment:
The `\n` at the end is displayed as text in my console, not sure why though 🤔 But maybe just add an empty echo if we want to have a break between this and the text below? Also I'd quote the format to make it stand out a bit.
```suggestion
  echo "Module name ${MODULE_NAME} not in recommended format \"github.com/repository/conduit-connector-connectorname\"."
  echo
```